src/components/ThumbnailNavigation.js

```diff
@@ -110,14 +121,39 @@ export class ThumbnailNavigation extends Component {
   /**
    * calculateScaledWidth - calculates the scaled width of a column for a Grid
    * in a simple case, a column == canvas. In a book view, a group (or two)
-   * canvases
+   * canvases. When in the "right" position, this value is static.
    */
   calculateScaledWidth(options) {
-    const { config, canvasGroupings } = this.props;
-    return canvasGroupings
-      .getCanvases(options.index)
-      .map(canvas => new ManifestoCanvas(canvas))
-      .reduce((acc, currentCanvas) => { return acc + (currentCanvas.hasValidDimensions ? Math.floor(config.thumbnailNavigation.height * currentCanvas.aspectRatio) : config.thumbnailNavigation.width); }, 8); // eslint-disable-line arrow-body-style, max-len
+    const { config, canvasGroupings, position } = this.props;
+    switch (position) {
+      case 'right':
+        return this.rightWidth();
+      // Default case bottom
+      default:
+        return canvasGroupings.getCanvases(options.index)
+          .map(canvas => new ManifestoCanvas(canvas))
+          .reduce((acc, currentCanvas) => { return acc + (currentCanvas.hasValidDimensions ? Math.floor(config.thumbnailNavigation.height * currentCanvas.aspectRatio) : config.thumbnailNavigation.width); }, 8); // eslint-disable-line arrow-body-style, max-len
+    }
+  }
+
+  /**
+   * calculateScaledHeight - calculates the scaled height of a row for a Grid
+   * in a simple case, a row == canvas. In a book view, a group (or two)
+   * canvases. When in the "bottom" position, this value is static.
+   */
+  calculateScaledHeight(options) {
+    const { config, canvasGroupings, position } = this.props;
+    switch (position) {
+      case 'right':
```

Comment:
I realize there is some repetitive case statements in may of these methods. I'm open to ideas about how to remove that or improve this. The idea here was to be pretty explicit so a reviewer/developer could understand what is happening.